app/tests/spec/lib/metrics.js

```diff
@@ -552,6 +552,16 @@ function (chai, $, p, Metrics, AuthErrors, Environment, sinon, _, WindowMock, Te
       });
     });
 
+    describe('logScreenEvent', function () {
+      beforeEach(function () {
```

Comment:
Don't think so, I didn't know that about `before`. Whatever gets the tests to pass is good with me. :)